catroid/src/main/res/values-ar/strings.xml

```diff
@@ -1447,4 +1447,14 @@
   <string name="crash_dialog_always_send">دائماً أرسل</string>
   <string name="crash_dialog_send">أرسل</string>
   <!--  -->
+
+  <!--  -->
+  <string name="Byte_short">بايت</string>
+  <string name="kiloByte_short">كيلوبايت</string>
```

Comment:
First! thanks for your replay 
actually this PR is no longer available, instant of 
" 84n4n4 commented on May 24 
This branch is 1 commit ahead, 89 commits behind Catrobat:develop.
this is not going to be mergable, please rebase and fix."
I deleted my repository because I didn't know how to fix that and I made a new fork. 
could you close this PR and have a look at the new PR under the Link : 
https://github.com/Catrobat/Catroid/pull/2333
>>>>>>>>>>>>>>>>>
it will be nice if we could meet tomorrow!